### PR TITLE
STUDY-305 Update project quickstart docs and get_patient_metadata_dataframe function

### PR DESCRIPTION
**Pull request by emilymahanna**
This PR updates the quickstart documentation for the project portion. It also updates the project patient dataframe function to change the column names to be clearer.
Screen Shot 2023-01-30 at 9 51 06 AM

### CODE REVIEW

**Review by gavin-philips — APPROVED** [review]
Niiiiiice

File: runeq/resources/project.py [hunk 1]
```diff
@@ -145,13 +145,26 @@ def get_patient_metadata_dataframe(self) -> pd.DataFrame:
...
+            "value": "metric_value",
+            "time_interval": "metric_time_interval",
+            "created_at": "metric_created_at",
+            "updated_at": "metric_updated_at",
+            "id": "metric_id",
+        }
         metadata_obj = self.to_dict()
-        metadata_df = pd.DataFrame(metadata_obj)
+        metadata_df = pd.DataFrame(metadata_obj).rename(
+            columns={"code_name": "project_code_name"}
```
Comment (abhinavpottabathula) [inline runeq/resources/project.py:159]: Just double checking here, the screenshot in the PR notes has values, for the `project_code_name` column, that are prefixed `patient-code-name-`. Is that supposed to be the case, or should the column be named `patient_code_name`?

**Review by abhinavpottabathula — CHANGES_REQUESTED** [review]
Just one question, but everything else looks good!

File: runeq/resources/project.py (hunk 1 above)
Comment (emilymahanna) [reply runeq/resources/project.py:159]: So the code names in this column are project specific, so it is a patient code name, but it is specific to the project (project code name). The screen shot just had the unfortunate coincidence of the project patient code name being `patient-code-name-114562`

Comment (emilymahanna) [reply runeq/resources/project.py:159]: does that make sense?

Comment (emilymahanna) [reply runeq/resources/project.py:159]: I could also label the column `project patient code name` to make it clear

Comment (abhinavpottabathula) [reply runeq/resources/project.py:159]: Ah got it, it's the patient's code name within the project. Yeah I think `project_patient_code name` would be better, but don't have strong feelings.

**Review by abhinavpottabathula — APPROVED** [review]
LGTM!

**Pull request merged**